website/client/components/shops/market/index.vue

```diff
@@ -187,6 +187,7 @@
                 .popover-content-text(v-html="$t('petLikeToEatText')", v-once)
 
         drawer-slider(
+          v-if="ownedItems(selectedDrawerItemType).length > 0"
```

Comment:
@negue this looks good to you?

Reply:
hmm @mrbeardy  could you also add `errorMessage` (like in this [example](https://github.com/HabitRPG/habitica/blob/develop/website/client/components/inventory/stable/index.vue#L165) ) to `drawer` that way the drawer isn't that empty ^^

Reply:
Can do :+1: